CDP4Composition/Converters/PanelViewConverter.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Globalization;
+using System.Reflection;
+using System.Text.RegularExpressions;
+using System.Windows;
+using System.Windows.Data;
+
+namespace CDP4Composition.Converters
+{
+    /// <summary>
+    /// Converter that resolves a panel view model to its view by name convention. 
+    /// The view is assumed to be in the same assembly as the view model.
+    /// </summary>
+    public class PanelViewConverter : IValueConverter
+    {
+        /// <inheritdoc/>
```

Comment:
Same comment as before, while we get the time-savings of inheritdoc, we do not use it.